examples/nlp/language_modeling/conf/megatron_gpt_prompt_learning_config.yaml

```diff
@@ -11,7 +11,7 @@ trainer:
   max_epochs: 3
   max_steps: null # consumed_samples = global_step * micro_batch_size * data_parallel_size * accumulate_grad_batches
   log_every_n_steps: 10
-  val_check_interval: 200
+  val_check_interval: 1.0
```

Comment:
why this is changed to 1?  Run val per 1 step?

Reply:
1.0 runs val once per epoch. I changed this because I made the CI tests use a smaller dataset and didn't want to worry about errors saying val check interval must be less than number of steps in 1 epoch.